Project.js

```diff
@@ -9,9 +9,12 @@ import {
   View
 } from 'react-native';
 import EStyleSheet from 'react-native-extended-stylesheet'
+import GoogleAnalytics from 'react-native-google-analytics-bridge'
 
 class Project extends Component {
   handleClick() {
+    GoogleAnalytics.trackEvent('View Project', this.props.project.display_name);
```

Comment:
Are you talking about the 'View Project'?  It looks like right now we don't have it very consistent on zooniverse global, but we can definitely define how we want this to look.  I'm fine with making it camelcase (just want to verify that's what you're talking about)

![image](https://cloud.githubusercontent.com/assets/1033477/18319714/82cdc244-74ec-11e6-9759-cd1a361e052f.png)